indico/modules/events/fields.py

```diff
@@ -121,72 +77,86 @@ class PersonLinkListFieldBase(EventPersonListField):
     default_sort_alpha = True
 
     widget = None
+    create_untrusted_persons = False
 
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super().__init__(*args, allow_groups=False, allow_external_users=True, **kwargs)
         self.object = getattr(kwargs['_form'], self.linked_object_attr, None)
+        self.event = self.object.event
```

Comment:
This is `None` when the event type is a lecture because no event attribute is present in the object.

```
AttributeError: 'NoneType' object has no attribute 'event
```